homeassistant/components/aurora/sensor.py

```diff
@@ -33,3 +33,8 @@ class AuroraSensor(AuroraEntity, SensorEntity):
     def native_value(self):
         """Return % chance the aurora is visible."""
         return self.coordinator.data
+
+    @property
+    def state_class(self):
+        """Return the state class of this entity, from STATE_CLASSES, if any."""
+        return SensorStateClass.MEASUREMENT
```

Comment:
Hi @emiltorp, the `state_class` property is not used anymore when it's constant. Better to add it as `_attr_state_class = SensorStateClass.MEASUREMENT`, near the current `_attr_native_unit_of_measurement` 👍

Reply:
Thanks. Good suggestion, updated the code and tested that it still works.